src/description.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var s = require('util').format;
+var marked = require('marked');
```

Comment:
> If we use readFile for file or string testing it means hitting the file system in both situations. Maybe not a big performance concern but still, feels a bit strange.

Then the only proper solution IMO is to have `description` and `descriptionFile` keys, or to make a configurable `filePattern`.